README.md

```diff
@@ -100,3 +100,9 @@ $ npm run lint
 ```
 
 Checks if the project has any linting errors.
+
+## Privacy Policy
+
+**Last update:** February 18, 2019.
```

Comment:
See my comment below: if we end up linking “here” to the privacy policy and include the Last Update in the document itself, I would remove this line.